mxcube3/ui/components/SampleView/MotorInput.js

```diff
@@ -65,6 +66,7 @@ export default class MotorInput extends React.Component {
                 step={step}
                 defaultValue={valueCropped}
                 name={motorName}
+                disabled={this.props.state === 4 || this.props.state === 3 || this.props.state <= 1}
```

Comment:
The problem I have found is that when the field is disabled it moves the cursor away from it and you will have to reselect it to move it again. 

Yes you can type in new data in the field while it is moving but it will be ignored and overwritten by the new value of the motorposition.

A possible solution would be to reselect the field for the user after the disable is removed..... This is probably better then the solution I implemented before.